modules/phase_field/src/materials/GrainAdvectionVelocity.C

```diff
@@ -23,9 +24,7 @@ InputParameters validParams<GrainAdvectionVelocity>()
 
 GrainAdvectionVelocity::GrainAdvectionVelocity(const InputParameters & parameters) :
    DerivativeMaterialInterface<Material>(parameters),
-   _grain_data(getUserObject<ComputeGrainCenterUserObject>("grain_data")),
-   _grain_volumes(_grain_data.getGrainVolumes()),
-   _grain_centers(_grain_data.getGrainCenters()),
+   _grain_tracker(getUserObject<GrainTrackerInterface>("grain_data")),
```

Comment:
We should standardize the parameter name for `GrainTrackerInterface` type parameters.